geopandas/io/arrow.py

```diff
@@ -8,7 +8,7 @@
 from geopandas.array import from_wkb, to_wkb
 from geopandas import GeoDataFrame
 import geopandas
-
+from geopandas.io.file import _expand_user
```

Comment:
```suggestion
from .file import _expand_user
```
This should be enough.